source/manual/load-test.html.md

```diff
@@ -0,0 +1,116 @@
+---
+owner_slack: "#govuk-developers"
+title: Run a load test with k6
+section: Testing
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These instructions are based on [this blog post](https://grafana.com/blog/2022/06/23/running-distributed-load-tests-on-kubernetes/).
+
+1. Log into AWS and set the context to the environment you wish to test:
+
+    ```sh
+    ENVIRONMENT=staging
+    ROLE=admin
+    AWS_REGION=eu-west-1
+
+    eval $(gds aws govuk-${ENVIRONMENT}-${ROLE} -e --art 8h)
+    kubectl config use-context $ENVIRONMENT
```

Comment:
**suggestion:** clarification on values

Looks like this assumes that you've named your context something like `staging`/`integation`/`production`. [This guidance](https://docs.publishing.service.gov.uk/kubernetes/get-started/access-eks-cluster/#access-a-cluster-for-the-first-time) suggests a format more like `govuk-staging`, although some people might have missed that advice. Might be worth linking to that guidance or something else in case anyone is confused about where these values come from and what they should be if this doesn't work for them